### ISSUE TITLE
virtualenv 14.0.4 fails on Windows (casing issue)

### ISSUE THREAD

**Issue by xflr6** [pypa/virtualenv#858]
```
C:\Users\User\Desktop>virtualenv venv
Traceback (most recent call last):
[...]
  File "c:\program files\python27\lib\site-packages\virtualenv.py", line 992, in
 change_prefix
    (filename, prefixes)
AssertionError: Filename C:\program files\python27\lib\os.py does not start with
 any of these prefixes: ['c:\\program files\\python27']
C:\Users\User\Desktop>virtualenv --version
14.0.4
```
see also #857, #856, #815

**Comment by The-Compiler** [comment]
I just ran into the same issue FWIW.

**Comment by Ivoz** [comment]
I released 14.0.5 for this

**Comment by The-Compiler** [comment]
Thanks! I can confirm it's fixed with 14.0.5.
